ckan/templates/package/resource_read.html

```diff
@@ -169,7 +169,7 @@ <h2>{{ _('Additional Information') }}</h2>
               </tr>
               {% for key, value in h.format_resource_items(res.items()) %}
                 {% if key not in ('created', 'metadata modified', 'last modified', 'format') %}
-                  <tr class="toggle-more"><th scope="row">{{ key | capitalize }}</th><td>{{ value }}</td></tr>
+                  <tr class="toggle-more"><th scope="row">{{ _(key | capitalize) }}</th><td>{{ value }}</td></tr>
```

Comment:
Resources can have arbitrarily-named extra fields added by users but translations are only added by site operators or administrators (it using the multilingual plugin). This could lead to surprising behavior if a user-provided extra field happened to match a translated English term.

Instead of a change like this in core consider overriding the `{% block resource_additional_information_inner %}` in the same extension where you add the download count to display the downloads label correctly.

Reply:
Another option is to use an extension like ckanext-scheming to define your custom resource field. ckanext-scheming handles overriding the resource information and showing labels in the correct language, you could use an `output_validator` to generate the downloads value instead of `before_resource_show` which would let the value be cached which should be much faster.

Reply:
> Resources can have arbitrarily-named extra fields added by users but translations are only added by site operators or administrators (it using the multilingual plugin). This could lead to surprising behavior if a user-provided extra field happened to match a translated English term.
> 
> Instead of a change like this in core consider overriding the `{% block resource_additional_information_inner %}` in the same extension where you add the download count to display the downloads label correctly.

Thank you for pointing that out. Could this issue also occur on the dataset page? On the dataset page, we have already implemented similar fixes as described in this PR.

Would it be possible to revise the approach to apply the same handling as on the dataset page?

Reply:
Yes ckanext-scheming can also be used to define custom dataset fields with labels translated into multiple languages.

I see that package extras *do* have their keys sent to gettext but unlike resource extras are not capitalized and are escaped first, which is even more inconsistent. My preference would be to remove the inconsistent behavior from both and perhaps give users a helper to format extra key names so that themes can override it however they see fit.

@amercader WDYT? Do you know if we're sending user data through gettext in other places (maybe this is intended behavior when using the multilingual plugin?)

Reply:
@wardi yes, this makes sense also Iooks like there is some broken `resource` property being set as well:

https://github.com/ckan/ckan/blob/328bf9e6146bf00fdca477127d47fe95c1e630fb/ckan/templates/package/snippets/additional_info.html#L85

Reply:
@wardi @amercader 
I understand that this PR addresses an important fix for translation issues in resource extra fields, but it seems that the discussion has stalled recently. Could you kindly provide an update on the current status or the next steps planned for this PR? Additionally, if further discussion is needed regarding how this PR impacts dataset pages or other related functionalities, please let me know. Thank you for your time and support.